content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -616,19 +728,20 @@ used and configured properly, you should see:
 
 ```shell
 u@node$ ifconfig cbr0 |grep PROMISC
+```
+```none
 UP BROADCAST RUNNING PROMISC MULTICAST  MTU:1460  Metric:1
-
 ```
 
 * Seek help if none of above works out.
```

Comment:
This advice could move into a What's next section. Does that seem like a good idea?

Reply:
I'm not clear on the concrete proposal?